grails-app/domain/org/pih/warehouse/inventory/TransactionType.groovy

```diff
@@ -39,4 +39,8 @@ class TransactionType implements Serializable {
         String regex = "\\${Constants.LOCALIZED_STRING_SEPARATOR}"
         return name.split(regex)[0] == transactionTypeName.split(regex)[0]
     }
+
+    Boolean isAdjustment() {
+        compareName("Adjustment")
```

Comment:
Let's move `"Adjustment"` to the const, and please add a comment here why it's determined like that.